src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/MetadataFileReferenceCompilationTests.vb

```diff
@@ -44,7 +44,10 @@ End Class
         Dim expectedErrors1 = <errors>
 BC31007: Unable to load module file 'Goo.dll': PE image doesn't contain managed metadata.
                  </errors>
-        CompilationUtils.AssertTheseDeclarationDiagnostics(compilation1, expectedErrors1)
+
+        Using New EnsureEnglishUICulture
```

Comment:
>Using New EnsureEnglishUICulture [](start = 8, length = 32)

The fact that you have to do this looks suspicious. I would expect AssertTheseDeclarationDiagnostics to take care of this

Reply:
The latter part of the message is controlled by the resource string `CodeAnalysisResources.PEImageDoesntContainManagedMetadata`. It's not added as a diagnostic argument but as the text of an exception. From PEModule.cs

```
throw new BadImageFormatException(CodeAnalysisResources.PEImageDoesntContainManagedMetadata);
```

Given it's an exception message I wouldn't expect the AssertTheseDeclarationDiagnostics method to be able to control it. Am I missing something? 



---
In reply to: [177594598](https://github.com/dotnet/roslyn/pull/25772#discussion_r177594598) [](ancestors = 177594598)

Reply:
Or scratch that, this is an thrown exception case 

---
In reply to: [177594598](https://github.com/dotnet/roslyn/pull/25772#discussion_r177594598) [](ancestors = 177594598)